egs/wsj/s5/steps/nnet3/dump_bottleneck_features.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+#           2016  Pegah Ghahremani
+# Apache 2.0
+# This script dumps bottleneck feature for model trained using nnet3.
+
+# Begin configuration section.
+feat_type=
+stage=1
+nj=4
+cmd=run.pl
+use_gpu=false
+bnf_name=renorm4
+use_ivector=false
+ivector_dir=
+# Begin configuration.
+transform_dir=
+# End configuration options.
+
+echo "$0 $@"  # Print the command line for logging
+
+[ -f path.sh ] && . ./path.sh # source the path.
+. parse_options.sh || exit 1;
+
+if [ $# != 5 ]; then
+   echo "usage: steps/nnet3/dump_bottleneck_features.sh <input-data-dir> <output-data-dir> <bnf-nnet-dir> <archive-dir> <log-dir>"
+   echo "e.g.:  steps/nnet3/dump_bottleneck_features.sh data/train data/train_bnf exp/nnet3/tdnn_bnf bnf exp_bnf/dump_bnf"
+   echo "main options (for others, see top of script file)"
+   echo "  --config <config-file>                           # config containing options"
+   echo "  --nj <nj>                                        # number of parallel jobs"
+   echo "  --cmd (utils/run.pl|utils/queue.pl <queue opts>) # how to run jobs."
+   echo "  --ivector-dir                                    # directory for ivectors"
+   exit 1;
+fi
+
+data=$1
+bnf_data=$2
+nnetdir=$3
+archivedir=$4
+dir=$5
+
+# Assume that final.nnet is in nnetdir
+cmvn_opts=`cat $nnetdir/cmvn_opts`;
+bnf_nnet=$nnetdir/final.raw
+if [ ! -f $bnf_nnet ] ; then
+  echo "No such file $bnf_nnet";
+  exit 1;
+fi
+
+if $use_gpu; then
+  train_queue_opt="--gpu 1"
```

Comment:
please rename these variables. We are not training or computing priors in this script.